release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1172,6 +1174,23 @@ In the following tables, features are marked with the following statuses:
 
 |====
 
+[discrete]
+=== OpenShift CLI (oc) deprecated and removed features
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.12 |4.13 |4.14
+
+|`--include-local-oci-catalogs` argument
+|Not Available
+|General Availability
+|Removed
+
+|`--use-oci-feature` argument
+|General Availability
+|Deprecated
+|Removed
+|====
```

Comment:
To be consistent with the usage across document, you could reword this as
`--include-local-oci-catalogs` parameter for `oc mirror`
`--use-oci-feature` parameter for `oc mirror`

Reply:
Thanks. Updated!